src/Filters/SimpleCollapsibleFilter.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+export default class SimpleCollapsibleFilter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showChildren: typeof props.showChildren === 'undefined' ? false : props.showChildren
```

Comment:
This line should be `showChildren: props.showChildren`

At the bottom, there should be something like:

```Javascript
CollapsibleFilter.propTypes = {
  showChildren: PropTypes.bool
}

CollapsibleFilter.defaultProps = {
  showChildren: true
}
```

This way, the default behavior is 'shown' but can be overridden, like is done in `SimpleFilter`